### PR TITLE
Feat/sui studio utils

### PR DESCRIPTION
**Pull request by carlesnunez**
# SUI-STUDIO-UTILS

Sui studio utils have the main purpose of englobe all the duplicated tools that we have on our SCM portals in one package with some improves.

## Description

The package is composed by two tools:

1. Domain builder

2. I18n by usecase and dictionary

More info about usage etc on the readme

### CODE REVIEW

File: packages/sui-studio-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  },
+  "keywords": [],
+  "author": "Enablers Frontend",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-cli": "6.26.0",
+    "babel-preset-sui": "1"
+  },
+  "dependencies": {
+    "@schibstedspain/rosetta": "^3.10.2"
```
Comment (carlosvillu) [inline packages/sui-studio-utils/package.json]: Use only the major.

**Comment by carlesnunez** [comment]
PR changes done @carlosvillu

File: packages/sui-studio-utils/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
+# sui-studio-tools
+> A set of sui-studio usable tools.
+
+```sh
+$ npm install @s-ui/sui-studio-tools --save
+```
+
+# Domain builder
+
+> Domain builder have the purpose of give to you the possibility of mock some of non implemented domain use cases meanwhile your team are developing them.
```
Comment (miduga) [inline packages/sui-studio-utils/README.md]: Domain builder *has* the purpose of *giving* you the possibility of *mocking*...

File: packages/sui-studio-utils/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+import myDomain from '@schibstedspain/myDomain'
+
+
+const domain = DomainBuilder.extend({ myDomain }).build()
+
+```
+
+### Mocking use cases
+
+> To mock use case you need to call two functions: 'for' and 'respondWith'
```
Comment (miduga) [inline packages/sui-studio-utils/README.md]: To mock `an` use case...

File: packages/sui-studio-utils/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+>
+> Lets supose that we want to mock a use case that isn't already implemented. It's name is 'get_products':
```
Comment (miduga) [inline packages/sui-studio-utils/README.md]: *Let's suppose*... *Its name is*...

File: packages/sui-studio-utils/package.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@s-ui/sui-studio-utils",
```
Comment (miduga) [inline packages/sui-studio-utils/package.json]: The name should be `@s-ui/studio-utils`.

**Pull request merged**